tests/periph_spi/main.c

```diff
@@ -91,14 +91,14 @@ int parse_spi_dev(int argc, char **argv)
         return -1;
     }
     if (argc >= 4) {
```

Comment:
Yes. If `argc` is "just" `>=4`  then "just" set `spi_mode` manually. If it's also `>=5` then also set `spi_speed` manually